src/cf-logstash-filters/src/logstash-filters/snippets/app-metric.conf

```diff
@@ -5,6 +5,7 @@
 if ( "" in [metrics_sd_id] ) {
   mutate {
     add_tag => [ "app_metric" ]
+    replace => { "@type" => "app_metric"}
```

Comment:
I think we should just call it `metric` ? Not all of the metrics are for apps., like the database metrics

Reply:
These are just the app metrics (container)